python/metrics/pyproject.toml

```diff
@@ -1,7 +1,41 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel",
-    "pip"
-]
+requires = ["setuptools>=61"]
```

Comment:
Lets bump this to `"setuptools>=70.0"`. See [CVE-2024-6345](https://nvd.nist.gov/vuln/detail/CVE-2024-6345) for motivation.

Reply:
Good catch!